func-complete-check/check-validator/index.ts

```diff
@@ -0,0 +1,28 @@
+import { AzureFunction, Context } from '@azure/functions'
+import { SubmittedCheckMessageV3, ValidateCheckMessageV1 } from '../typings/message-schemas'
+import { performance } from 'perf_hooks'
+const functionName = 'check-validator'
+import V1 from './v1'
+
+const serviceBusQueueTrigger: AzureFunction = async function (context: Context, validateCheckMessage: ValidateCheckMessageV1): Promise<void> {
+  const start = performance.now()
+  const version = validateCheckMessage.version
+  context.log.info(`${functionName}: version:${version} message received for checkCode ${validateCheckMessage.checkCode}`)
+  try {
+    if (version !== '1') {

```

Comment:
Version might be better suited to being an int

Reply:
i tried this, but it doesn't work. (hence the typeof logging above)
when the message is injected into the function via the binding it sets it to a string.
so when it does the check for version, it always fails.
i will leave the interface types as numbers to express our intent, but will keep the comparison as a string to ensure it works.  rubbish 🦆